packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1981,6 +2010,46 @@ export default class Meeting extends StatelessWebexPlugin {
       });
   }
 
+  /**
+   * Checks if the supplied registrationId is correct. It returns a promise with information whether the
+   * password and captcha code were correct or not.
+   * @param {String} registrationId - can be undefined if only captcha was required
```

Comment:
the comment says it can be undefined, but the TS type says it has to be a string